docs/docs/blocks/!/Ref.md

```diff
@@ -0,0 +1,42 @@
+# Ref
+
+```clojure
+(Ref
+  :Name [(String) (ContextVar [(Any)])]
+  :Key [(String) (ContextVar [(String)]) (None)]
+  :Global [(Bool)]
+  :Serialized [(Bool)]
+)
+```
+
+## Definition
+
+
+## Parameters
+| Name | Type | Default | Description |
+|------|------|---------|-------------|
+| Name | `[(String) (ContextVar [(Any)])]` | `""` | The name of the variable. |
+| Key | `[(String) (ContextVar [(String)]) (None)]` | `None` | The key of the value to read/write from/in the table (this variable will become a table). |
+| Global | `[(Bool)]` | `false` | If the variable is or should be available to all of the chains in the same node. |
+| Serialized | `[(Bool)]` | `false` | If the variable should be serialized as chain variable when the parent chain is serialized, e.g. (ToBytes), this option is mostly to be used with (Once) blocks with serialization turned off or (Get) blocks with default values set. |
+
+
+## Input
+| Type | Description |
+|------|-------------|
+| `[(Any)]` |  |
+
+
+## Output
+| Type | Description |
+|------|-------------|
+| `[(Any)]` |  |
+
+
+## Examples
+
+```clojure
```

Comment:
Btw I noticed mkdocs has no clojure syntax highlight, we should investigate how to include that (pretty sure it's probably a flag or some modules to add)

Reply:
There are plugins to add syntax coloring. It just that the documentation of such plugins is very hard to understand: they expect you to know how mkdocs works internally and provide no easy step to integrate it.

Reply:
Made an issue